opm/simulators/linalg/bda/WellContributionsOCLContainer.cpp

```diff
@@ -79,8 +91,8 @@ namespace bda
         }
 
         if(!wellContribs.multisegments.empty()){
-            multisegments = std::move(wellContribs.multisegments);
-            num_ms_wells = multisegments.size();
+            multisegments = std::move(std::make_unique<mswVecT>(wellContribs.multisegments));
```

Comment:
This is now a change of semantics, not sure whether we really want that.

Previously we moved the vector from `wellContribs` to here. Now we seem to make a copy of the vector in `wellContribs` just to put it in a `std::unique_ptr`.